### PR TITLE
Add Variables and imporve naming

### PR DESCRIPTION
**Pull request by bmacharia**

### CODE REVIEW

File: terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,7 @@
+env_code = "main"
+
+vpc_cidr = "10.0.0.0/16"
+
+public_cidr = ["10.0.0.0/24", "10.0.1.0/24"]
+
+private_cidr = ["10.0.2.0/24", "10.0.3.0/24"]
```
Comment (vladimirmukhin) [inline terraform.tfvars]: Please add a newline to the end of this file

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+variable "env_code" {
+  type = string
+}
+
+variable vpc_cidr {}
+
+variable  private_cidr {}
+
+variable public_cidr {}
```
Comment (vladimirmukhin) [inline variables.tf]: Please add a newline to the end of this file

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: terraform.tfvars (hunk 1 above)
Comment (bmacharia) [reply terraform.tfvars]: newline as been appended to file terraform.tfvars

File: variables.tf (hunk 2 above)
Comment (bmacharia) [reply variables.tf]: newline has been appended to variables.tf file

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**